translations/harbour-osmscout-server-es.ts

```diff
@@ -230,47 +230,47 @@
     <message>
         <location filename="../src/routingforhuman.cpp" line="98"/>
         <source>Drive along %along%</source>
-        <translation>Continua por la %along%</translation>
+        <translation>Conducir a lo largo%a lo largo%</translation>
```

Comment:
%along% should not be translated. See README at https://github.com/rinigus/osmscout-server/blob/master/translations/README